descartes_core/include/descartes_core/robot_model.h

```diff
@@ -102,6 +102,17 @@ class RobotModel
   virtual bool initialize(const std::string robot_description, const std::string& group_name,
                           const std::string& world_frame,const std::string& tcp_frame) = 0;
 
+  /**
+   * @brief Performs necessary checks to see if the robot is capable of moving from the initial joint pose
+   *        to the final pose in dt seconds
+   * @param  from_joint_pose [description]
+   * @param  to_joint_pose   [description]
+   * @param  dt              [description]
+   * @return                 [description]
+   */
+  virtual bool isValidMove(const std::vector<double>& from_joint_pose, const std::vector<double>& to_joint_pose,
```

Comment:
Should this method name be more specific?  It really only checks velocity limits?  The name implies more.